### ISSUE TITLE
Telnet admins can use /onlinemode (Setting onlinemode True / False)

### ISSUE THREAD

**Issue by Wild1145** [TotalFreedom/TotalFreedomMod#1972]
It appears that telnet admins can set onlinemode to true and false, when the latest code on branch is showing that you should need a senior admin.

**Comment by ghost** [comment]
This is an easy fix, but hoping not to offend anyone in saying this, the code for the command looks like someone puked and tried to clean it up.

**Comment by ghost** [comment]
@Wild1145 is not a bug, mark allowed us to set it, due to most seniors aren't active daily, most are telnet. soo keep it , we aren't abusing of onlinemode.


plus spambots can't be blocked just because online seniors are most rare, most telnet are on, and they are using onlinemode to block them.

**Comment by ExactObject** [comment]
No, that's by design.  We don't have seniors on enough and the need to change modes is constant given attacks.  Thanks.

**Comment by Wild1145** [comment]
It is still classed as a bug. The codebase is showing it should be seniors only. Be it by design or not, it is an unintended feature of the TFM plugin.

**Comment by ExactObject** [comment]
Let's change the codebase than.

**Comment by Mafrans** [comment]
Change the codebase in what aspect, set it to Telnet Console or fix it so only Seniors can use it

**Comment by Wild1145** [comment]
Well if someone on the server has changed this, it should be raised in the near future here as a patch on the server that is not here. If it is then we will change it if it is felt that it is the most appropriate way forward, or if it is not the most appropriate way forward or not raised here, we will keep it as senior admin.

**Comment by ghost** [comment]
Mark said we keep as telnet admins...

**Comment by Wild1145** [comment]
@marcocorriero Then it needs to be raised as per my previous comment. Until such a time, it will be treated as a bug as that is currently what it is.
